packages/components/radio/src/radio.css

```diff
@@ -0,0 +1,87 @@
+/* General........................................................ */
+
+:host {
+	& label {
+		display: block;
+		position: relative;
+		padding-left: var(--ts-unit-plus);
+		margin: var(--ts-unit-half);
+		cursor: pointer;
+		font-size: var(--ts-fontsize);
+		-webkit-user-select: none;
+		-moz-user-select: none;
+		-ms-user-select: none;
+		user-select: none;
+
+		&:hover {
+			& span {
+				background-color: var(--ts-color-gray-lighter);
+			}
+		}
+
+		& input {
+			position: absolute;
+			opacity: 0;
+			cursor: pointer;
+			height: 0;
+			width: 0;
+		}
+
+		& span {
```

Comment:
Please use class names, it make it easier to understand the styles. 
This is a topic that we need to discuss later in team and add it to the guideline.

Reply:
Sure. I thought we prefer to use attributes, properties, and structure over class names (for example button element).  But It could be better to use move descriptive class names where it is possible.

Do we have any special naming rules for classes? I suppose I would call it "checkbox" or "flag".

Reply:
Fixed

Reply:
> I thought we prefer to use attributes, properties, and structure over class names (for example button element). But It could be better to use move descriptive class names where it is possible.

Yes, when there's an attribute or property that already have a clear name then that should be enough, but using tag name as selector make it hard to understand what it does!

>Do we have any special naming rules for classes? 
No, at least not yet!